bitrise.yml

```diff
@@ -27,6 +27,21 @@ workflows:
     - ci-parallel
     - tvos
   
+  manual_test-retry_after_fail:
```

Comment:
Is it possible to add it to the `ci` workflow's after_run list?

Reply:
Possible, but not enough, as check of exit code would be needed.